public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -1,5 +1,5 @@
-import { cloneDeep, merge, isEqual } from 'lodash';
```

Comment:
wow, great catch 🎉 but you changed this before and reverted it 🤔 
https://github.com/grafana/grafana/pull/96881
https://github.com/grafana/grafana/pull/96999

Reply:
i addressed this in the PR description. the prior attempt just swapped out the function without changing anything else.

the same dashboards that broke last time work correctly here.